src/response/hessian.jl

```diff
@@ -175,17 +175,19 @@ function solve_ΩplusK_split(ham::Hamiltonian, ρ::AbstractArray{T}, ψ, occupat
         end
     end
 
-    δψ = apply_χ0_4P(ham, ψ, occupation, εF, eigenvalues, δHψ;
-                     occupation_threshold, abstol=tol_sternheimer, reltol=0, kwargs...)
-    (; δψ, δρ, δHψ, δVind, δeigenvalues, history)
+    δψ, δoccupation, δεF = apply_χ0_4P(ham, ψ, occupation, εF, eigenvalues, δHψ;
+                                       occupation_threshold, abstol=tol_sternheimer, 
+                                       reltol=0, kwargs...)
+
+    (; δψ, δρ, δHψ, δVind, δeigenvalues, δoccupation, δεF, history)
```

Comment:
if you return a named tuple you can merge here.

Reply:
Agree, though for only three args returned from apply_χ0_4P the explicit names could be more readable?

Reply:
I think they're ok.